en/guide/mavlink_2.md

```diff
@@ -1,28 +1,72 @@
 # MAVLink 2
 
-MAVLink version 2 is a backward-compatible new version. The main new features are:
+*MAVLink 2* is a backward-compatible update to the MAVLink protocol that has been designed to bring more flexibility and security to MAVLink communication.
 
-* Support for more than 256 message IDs \(now 24 bit or over 16 million packets\)
-* Packet signing \(authentication\)
-* Support for extending existing messages
-* Support for variable length arrays
+The key new features of *MAVLink 2* are:
+* More than 256 message IDs (24 bit message ID - over 16 million packets)
+* [Packet signing](../guide/message_signing.md) (authentication)
+* [Extending existing MAVLink messages](#message_extensions)
+* [Variable length arrays](#variable_arrays) (*simplified implementation*).
 
+*MAVLink 2* bindings have been developed for C and Python (other languages are not implemented yet).
```

Comment:
Supported languages section https://mavlink.io/en/#supported_languages says that only C and C++ 11 support Mavlink2. Is Python also supported?

Reply:
@LorenzMeier Can you confirm the above?

Reply:
C and Python are definitely supported. All other languages have to be vetted one-by-one.